### PR TITLE
Adds in support docs for 0.3 release

### LINKED ISSUE #1158: [FEA] Better documentation on type support
**Is your feature request related to a problem? Please describe.**
It is often not clear from reading the documentation which types are supported and which are not.  Often this is the first step in determining whether a query may benefit substantially by using the RAPIDS Accelerator for Apache Spark.

**Describe the solution you'd like**
The documentation should have a section describing which types are supported and general discussion on the level of support per type (e.g.: IntegerType supported in every operator the plugin supports, while other types may only have partial coverage across operators).

**Describe alternatives you've considered**
We could update the section on operations supported to include a list of types supported for that specific operation.

### PR DESCRIPTION
**Pull request by revans2**
Code changes to generate the docs will go into 0.4

This fixes #1158 although the actual document generation will come later on.

### CODE REVIEW

**Comment by revans2** [comment]
build

File: docs/supported_ops.md [hunk 1]
```diff
@@ -0,0 +1,15883 @@
...
+<td><b>NS</b></td>
+<td><b>NS</b></td>
+<td><b>NS</b></td>
+</tr>
+</table>
+* as was state previously Decimal is only supported up to a precision of 
+18 and Timestamp is only supported in the 
+UTC time zone.
+
+# `Expression` or functions:
```
Comment (jlowe) [inline docs/supported_ops.md:728]:
```suggestion
# `Expression` or Functions
```

**Comment by revans2** [comment]
build

**Comment by revans2** [comment]
@jlowe please take another look

**Review by jlowe — APPROVED** [review]

**Pull request merged**